### DISCUSSION TITLE
Change logging in production

### DISCUSSION THREAD

**Discussion by ProfBrickz** [csmith1188/Formbar.js#296] [Ideas]
Should we configure Winston to log selectively based on the environment variable in production? This could streamline logs by capturing only warnings, errors, and critical issues, reducing clutter in the production environment. However, there's a potential downside of limited logs hindering debugging in the production stage due to reduced information. What are your thoughts on this approach?

**Comment by csmith1188** [comment]
This is a good idea, but we would still need a log for non-warnings/errors to determine if the formbar was misused and how and by who.

**Comment by APlitt24235** [comment]
This is definitely possible, and it would help streamline things for sure. However, we'd need to determine what exact information we should include and exclude. I believe that all error checking should still be in place no matter the build in case something unprecedented were to occur.
